live-examples/html-examples/input/css/input.css

```diff
@@ -0,0 +1,51 @@
+@font-face {
+    font-family: 'Fira Sans';
+    src: local('FiraSans-Regular'),
+    url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+}
+
+legend {
+    background-color: #000;
+    color: #fff;
+    padding: 3px 6px;
+}
+
+input,
+label {
+    width: 43%;
+}
+
+input {
+    margin: .5rem 0;
+    padding: .5rem;
+    border-radius: 4px;
+    border: 1px solid #ddd;
+}
+
+label {
+    display: inline-block;
+    font-size: .8rem;
+}
+
+input:invalid + span:after {
+    content: '✖';
+    color: red;
+    padding-left: 5px;
+}
+
+input:valid + span:after {
+    content: '✓';
+    color: green;
+    padding-left: 5px;
+}
+
+.warning {
+    font-size: .65rem;
+    color: #e67d09;
+}
+
```

Comment:
@schalkneethling Removed outline: none, 0.5rem => .5rem, spacing fixes, #ffffff => #fff in CSS